packages/plugin-core/src/commands/Goto.ts

```diff
@@ -1,25 +1,36 @@
 import {
   DendronError,
   DVault,
+  ERROR_STATUS,
   NoteUtils,
   RespV3,
   VaultUtils,
 } from "@dendronhq/common-all";
+import { resolvePath, vault2Path } from "@dendronhq/common-server";
+import fs from "fs";
 import _ from "lodash";
+import open from "open";
+import path from "path";
+import { env, Uri } from "vscode";
+import { PickerUtilsV2 } from "../components/lookup/utils";
 import { DENDRON_COMMANDS } from "../constants";
 import { IDendronExtension } from "../dendronExtensionInterface";
 import { ExtensionProvider } from "../ExtensionProvider";
 import { getLinkFromSelectionWithWorkspace } from "../utils/editor";
+import { getURLAt } from "../utils/md";
+import { VSCodeUtils } from "../vsCodeUtils";
+import { getDWorkspace, getExtension } from "../workspace";
 import { BasicCommand } from "./base";
 import { GotoNoteCommand } from "./GotoNote";
 import { GoToNoteCommandOutput, TargetKind } from "./GoToNoteInterface";
 import { OpenLinkCommand } from "./OpenLink";
 
 type CommandOpts = {};
 
-type CommandInput = {};
-
-type CommandOutput = RespV3<GoToNoteCommandOutput>;
+type CommandOutput =
+  | RespV3<GoToNoteCommandOutput>
+  | { error?: DendronError; fsPath?: string }
```

Comment:
instead of copying the output of `OpenLink`, can we use `GoToNoteCommandOutput`? 

{ kind: TargetKind.LINK; fullPath: string; fromProxy: boolean }